testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -2595,7 +2595,7 @@ void checkModOperatorDivByZero(SqlOperatorFixture f) {
     }
     f.checkNull("1e1 / cast(null as float)");
 
-    f.checkScalarExact("100.1 / 0.00000000000000001", "DECIMAL(19, 0) NOT NULL",
+    f.checkScalarExact("100.1 / 0.00000000000000001", "DECIMAL(19, 19) NOT NULL",
```

Comment:
This looks wrong, DECIMAL(19, 19) cannot represent the result of this computation.
Mysql infers a result of DECIMAL(25, 5) for the result.

Reply:
I'll take a look, thanks for pointing it out. Do you've any guidance, which vendors/dbs one should try? (which do you usually try?) I guess some deviate more than others from the standard.

Reply:
Postgres is the most referenced one in discussions here around, but it's generally a good practice to check SQL Server, Oracle, MySQL as well (https://www.db-fiddle.com/ and https://sqlfiddle.com/ are a convenient way to do so in general).

Reply:
In this case Postgres is not a good reference, because it supports arbitrary precision DECIMAL numbers. I have found a relatively precise description for SQL server and I will submit an alternate PR for this issue.